### PR TITLE
Update somaiyalbakhet.json

### PR DESCRIPTION
**Pull request by Somaiya-XI**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://somaiya.netlify.app/

### CODE REVIEW

File: somaiyalbakhet.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "record": {
         "A": [
             "75.2.60.5"
-        ]
+        ],
+        "CNAME": "somaiya.netlify.app"
```
Comment (CuteDog5695) [inline somaiyalbakhet.json:11]: Can't have CNAME with A records, pick either of them.

File: somaiyalbakhet.json
Comment (CuteDog5695) [inline somaiyalbakhet.json:1]: File isn't in the `domains` directory.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: somaiyalbakhet.json (hunk 1 above)
Comment (Somaiya-XI) [reply somaiyalbakhet.json]: ok removed CNAME, I thought it has to be there after merging the main Netlify fallback A record

**Review by iostpa — APPROVED** [review]
I just noticed that the file wasn't in the domain directory yet it got merged, HOWWWWWWWWWWWWWWWWWWWW.
Anyways, I'll merge this PR now to fix the problem.

**Comment by CuteDog5695** [comment]
Fuck, admin has to merge this PR, please wait for a bit.

**Pull request merged**